### PR TITLE
Remove success screen at end of admin creation journey

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
We're replacing most of the success pages with notification banners. This is that for the admin creation journey

### Changes proposed in this pull request
- Replace success panel with notification banner
- Make use of data-test attributes in a couple of places

### Testing
Updated existing

### CODE REVIEW

**Review by amazimbe — APPROVED** [review]

File: app/controllers/admin/administrators/administrators_controller.rb [hunk 1]
```diff
@@ -49,6 +49,9 @@ def create
           AdminProfile.create!(user: user)
         end
         session.delete(:administrator_user)
+
+        set_success_message(heading: "User added", content: "They have been sent an email to sign in")
```
Comment (kicferk1) [inline app/controllers/admin/administrators/administrators_controller.rb:53]: Is this true? Or make belief?

**Review by kicferk1 — APPROVED** [review]

File: app/controllers/admin/administrators/administrators_controller.rb (hunk 1 above)
Comment (seb-siem-softwire) [reply app/controllers/admin/administrators/administrators_controller.rb:53]: Have you double checked the message with Shani?

File: app/views/admin/administrators/administrators/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <h1 class="govuk-heading-l">Administrators</h1>
-<%= govuk_link_to "Create a new administrator", new_admin_administrator_path, button: true %>
+<%= govuk_link_to "Create a new administrator", new_admin_administrator_path, button: true, "data-test" => "create-admin-button" %>
```
Comment (seb-siem-softwire) [inline app/views/admin/administrators/administrators/index.html.erb:2]: Nice - there is however a problem with submit button just to be aware - a workaround is to simply add a class when working with submit buttons

**Review by seb-siem-softwire — APPROVED** [review]
Couple of small things but otherwise good - Double check with Shani if the wording is correct

File: app/views/admin/administrators/administrators/index.html.erb (hunk 2 above)
Comment (callumacrae) [reply app/views/admin/administrators/administrators/index.html.erb:2]: Sending a PR fixing that today! :)

File: app/controllers/admin/administrators/administrators_controller.rb (hunk 1 above)
Comment (ethanmills) [reply app/controllers/admin/administrators/administrators_controller.rb:53]: This is Shani's message. Alfred is implementing this. Once we know when we will be sending emails to lead providers I will get some updated copy for that one

**Pull request merged**